client/src/components/common/starRating.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-regular-svg-icons';
+import { faStar as faSolidStar, faStarHalfAlt} from '@fortawesome/free-solid-svg-icons';
+import CSS from './starRating.module.css';
+
+const StarRating = () => {
+  return (
+    <div>
+      <FontAwesomeIcon icon={faSolidStar} className={CSS['star-rating']} />
+      <FontAwesomeIcon icon={faSolidStar} className={CSS['star-rating']} />
+      <FontAwesomeIcon icon={faSolidStar} className={CSS['star-rating']} />
+      <FontAwesomeIcon icon={faStarHalfAlt} className={CSS['star-rating']} />
+      <FontAwesomeIcon icon={faStar} className={CSS['star-rating']} />
+    </div>
+  )
+}
+
+export default StarRating;
```

Comment:
Code looks good. I really gotta look into the css module thing cause that looks pretty nifty. For fontawesome I've been using a library in its own file, do you think that'd be useful? We wouldn't have to worry about importing each icon into every file we need them in. This is what I've been doing:

```suggestion
// this is in src/components/fontawesome.js
import { library } from '@fortawesome/fontawesome-svg-core';

import { faArrowRight, faArrowLeft, faAngleDown, faAngleUp,  } from '@fortawesome/free-solid-svg-icons';

library.add(
  faArrowRight,
  faArrowLeft,
  faAngleDown,
  faAngleUp,

);
```

And because we wouldn't be importing the specific icon we'd define them this way:
`<FontAwesomeIcon icon={'arrow-right'} />`

Reply:
Makes sense! It's nice to have a central hub for those icons! I made a `common` folder to keep our reusable components, and I put the icon file there as well.